data/reusables/project-management/access-collaboration-settings.md

```diff
@@ -1,2 +1,2 @@
-1. Click {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %}, then click **Settings**.
-![Settings option in drop-down menu from project board sidebar](/assets/images/help/projects/project-board-settings.png)
+1. Click {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %}, then click **Settings**. You won't see this if you are not an org owner or project admin.
```

Comment:
We shouldn't be introducing permissions information in the middle of a procedure like this. Instead, we should use permission statements at the beginning of each procedure that requires this step. 

For more information about permissions statements, see [the content models](https://github.com/github/docs/blob/main/contributing/content-model.md#permissions-statements). You should be able to search the repository for all uses of this reusable to find all the procedures that need these permissions statements.

Reply:
Hey @lecoursen as per your suggestion, I shifted the permission statement at the start of the procedure.